src/qt/rpcconsole.cpp

```diff
@@ -574,6 +574,7 @@ bool RPCConsole::eventFilter(QObject* obj, QEvent *event)
             if(obj == autoCompleter->popup()) {
                 autoCompleter->popup()->hide();
                 QApplication::postEvent(ui->lineEdit, new QKeyEvent(*keyevt));
+                autoCompleter->popup()->hide();
```

Comment:
(14450) this was fixed via https://github.com/dashpay/dash/pull/1898 already. If you want this to be merged purely to align code/history with bitcoin you should drop line 575.